api-samples/identity/README.md

```diff
@@ -0,0 +1,23 @@
+<a target="_blank" href="https://chrome.google.com/webstore/detail/oficlfehfenioickohognhdhmmcpceil">![Try it now in CWS](https://raw.github.com/GoogleChrome/chrome-extensions-samples/master/apps/tryitnowbutton.png "Click here to install this sample from the Chrome Web Store")</a>
+
+
+# Identity
```

Comment:
Can we follow the README template in the root of the repository? In particular, it would be good to describe how to load the extension and what buttons you are expected to press.